index.js

```diff
@@ -0,0 +1,9 @@
+require('dotenv').config()
+
+const server = require('./server/server');
+
+PORT = process.env.PORT || 4500
```

Comment:
Missing a `const` here
```suggestion
const PORT = process.env.PORT || 4500
```